src/main/java/com/bulkbuy/enterprise/controller/NavigationController.java

```diff
@@ -55,7 +55,7 @@ public Order saveOrder(@RequestBody Order order) throws Exception {
     }
 
     @RequestMapping("/orderLookup")
-    public String orderLookup(){
+    public String lookupOrder(){
```

Comment:
Method names should start with the verb
https://www.baeldung.com/java-clean-code